test/uk/gov/hmrc/economiccrimelevyaccount/connectors/DesConnectorSpec.scala

```diff
@@ -18,56 +18,42 @@ package uk.gov.hmrc.economiccrimelevyaccount.connectors
 
 import org.mockito.ArgumentMatchers
 import org.mockito.ArgumentMatchers.any
-import play.api.http.HeaderNames
+import play.api.libs.json.Json
 import uk.gov.hmrc.economiccrimelevyaccount.base.SpecBase
-import uk.gov.hmrc.economiccrimelevyaccount.generators.CachedArbitraries.arbOptObligationData
-import uk.gov.hmrc.economiccrimelevyaccount.models.CustomHeaderNames
+import uk.gov.hmrc.economiccrimelevyaccount.generators.CachedArbitraries._
+import uk.gov.hmrc.economiccrimelevyaccount.models.EclReference
 import uk.gov.hmrc.economiccrimelevyaccount.models.des.ObligationData
-import uk.gov.hmrc.economiccrimelevyaccount.utils.CorrelationIdGenerator
-import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.{HttpResponse, StringContextOps}
+import uk.gov.hmrc.http.client.{HttpClientV2, RequestBuilder}
 
 import java.time.{LocalDate, ZoneOffset}
 import scala.concurrent.Future
 
 class DesConnectorSpec extends SpecBase {
```

Comment:
Retries should be confirmed in this spec for the Des connector.

See example:

https://github.com/hmrc/economic-crime-levy-registration/blob/944f6bd1e935ec0ec1883d5684d6c4e4e760e06f/test/uk/gov/hmrc/economiccrimelevyregistration/connectors/DmsConnectorSpec.scala#L81